index.html

```diff
@@ -0,0 +1,273 @@
+                    </li>
+                    <li class="mobile-link">
+                        <a href="#">About</a>
+                    </li>
+                    <li class="mobile-link">
+                        <a href="#">Contact</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+
+    <section class="head-section">
+        <div id="head-container">
+            <h1 class="head">
+                I'm Edith, <br> 
+                Glad to see you!
+            </h1>
+            <p class="desc">I’m a software developer! I can help you build a product, feature or website
+                Look through some of my work and experience. If you like what you see and have a edith you need coded, 
+                don’t hesitate to contact me!
+            </p>
+            <h2 class="connect">LET’S CONNECT</h2>
+
+            <ul class="social-media-links">
+                <li> <a href="#" target="_blank" rel="noopener" class="media-link"><i
+                    class="fa-brands fa-twitter fa-2x"></i> </a> </li>
+                <li> <a href="#" target="_blank" rel="noopener"
+                    class="media-link"><i class="fa-brands fa-linkedin fa-2x"></i> </a> </li>
+                <li> <a href="#" target="_blank" rel="noopener" class="media-link"><i
+                    class="fa-brands fa-github fa-2x"></i> </a></li>
+                <li> <a href="#" target="_blank" rel="noopener"
+                    class="media-link"><i class="fa-brands fa-hackerrank fa-2x"></i> </a></li>
+            </ul>
+        </div>
+    </section>
+
+
+    <main class="grid-container">
+        <section class="works-card-tonic">
+            <img class="snapshort tonic-img" src="./assets/tonic.png" alt="snapshoot">
+            <div class="left-block-tonic">
+                <div class="primary-container-tonic">
+                    <h1 class="edith-title-tonic">Tonic</h1>
+                    <div class="newme2-tonic">
+                        <div class="client"><h3>CANOPY</h3></div>
+                        <img src="./assets/Counter.png" alt="counter image">
+                        <div class="role"><h3>Back End Dev</h3></div>
+                        <img src="./assets/Counter.png" alt="counter image" class="dot">
+                        <div class="year"><h3> 2015</h3></div>
+                    </div>
+                </div>
+                <p class="edith-desc-tonic">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="plangs-tonic">
+                    <li class="plang1"><p>html</p></li>
+                    <li class="plang2"><p>css</p></li>
+                    <li class="plang3"><p>javascript</p></li>
+                </ul>
+                <form action="https://github.com/vigehi"  class="ediths-tonic">
+                    <button type="submit" class="go-ediths">See ediths</button>
```

Comment:
- [ ] I think it would be better to change the button text to `See Project`  to be more descriptive 👌 You must replicate the design as much as possible (e.g., font, colors, images, `text`, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- [ ] Remember to implement this change in all buttons you have 💯👌

- [ ] I also suggest using the same interactive buttons based on your Figma design 👌

![image](https://user-images.githubusercontent.com/45231276/179211803-932af9b0-94fa-4911-9741-ea0a580f3187.png)